packages/ecdh/tests/ecdh.test.ts

```diff
@@ -0,0 +1,57 @@
+import { BarretenbergBackend } from '@noir-lang/backend_barretenberg';
+import { Noir } from '@noir-lang/noir_js';
+import { ProofData } from '@noir-lang/types';
+import { expect } from 'chai';
+import { randomBytes } from 'crypto';
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
+import 'mocha';
+import circuit_ecdh from '../../../target/ecdh.json';
+
+function generatePrivateKey(): Uint8Array {
+  return randomBytes(32);
+}
+
+describe('ECDH Circuit Tests', function() {
+  let noir: Noir;
+  let backend: BarretenbergBackend;
+  let correctProof: ProofData;
+
+  beforeEach(async () => {
+    const circuitFile = readFileSync(resolve(__dirname, '../../../target/ecdh.json'), 'utf-8');
+    const circuit = JSON.parse(circuitFile);
+    backend = new BarretenbergBackend(circuit);
+    noir = new Noir(circuit, backend);
```

Comment:
aren't we reading/loading the same file twice here?
```suggestion
import { CompiledCircuit, ProofData } from '@noir-lang/types';
import { expect } from 'chai';
import { randomBytes } from 'crypto';
import { readFileSync } from 'fs';
import { resolve } from 'path';
import 'mocha';
import circuit from '../../../target/ecdh.json';

function generatePrivateKey(): Uint8Array {
  return randomBytes(32);
}

describe('ECDH Circuit Tests', function() {
  let noir: Noir;
  let backend: BarretenbergBackend;
  let correctProof: ProofData;

  beforeEach(async () => {
    backend = new BarretenbergBackend(circuit as CompiledCircuit);
    noir = new Noir(circuit as CompiledCircuit, backend);
```

We need to use type assertions to tell typescript about the type of the JSON we are importing